### DISCUSSION TITLE
Asset Outreach Team

### DISCUSSION THREAD

**Discussion by morgajel** [godotengine/godot-proposals#8028] [General Discussions]
(Apologies if I make any mistakes here, I'm new to Godot myself.)

### The Problem
Godot currently lags behind its counterparts when it comes to asset packages in their respective asset libraries; simply compare Godot's AssetLib to Unity's AssetStore. Many of the assets in the Unity Store are free, with many of them having friendly licensing and public git repositories.  I think Godot may be able to capitalize on its newfound visibility and convince some of these asset creators to support Godot in addition to existing platforms. 

### The Proposal

I'd like to propose creating an Asset Outreach Team.

### Purpose

The purpose of the Asset Outreach Team would be to:

- Create a set of criteria that asset packages must meet to be considered (acceptable licensing, cost, technology used, etc);
- Curate a list of popular asset packages from other platforms to investigate;
- Identify high-value asset packages and prolific authors that may be open to supporting Godot;
- Contacting Authors on behalf of the Godot community to see if they would be willing to add Godot support;
- Writing easy-to-use CI/CD directions to add Godot support and publish in the Godot Asset Library;
- Assist authors in adding Godot Asset Library support where needed;

The initial focus should be on simple assets- models, textures, sounds, etc.  If this proves viable and successful, other asset types can be investigated.

### Pitfalls

- This should be a well-coordinated effort to avoid overwhelming creators and/or giving the Godot community a whiny/demanding reputation.
- This should not require nor impact core developers- their time is precious and should be spent on improving Godot.
- Finding people knowledgeable and technical enough to help authors may prove difficult in the long run.

Thoughts?

**Comment by adamscott** [comment]
The Godot Foundation is currently actively working on overhauling the asset store, to permit the sale of plugins and assets.

I think it's a prerequisite of such a team. Because we are not able currently to offer paid authors a way to make money with Godot.
